modules/storage-aws-s3/s3/s3.go

```diff
@@ -102,7 +110,7 @@ func (s *s3) StoreSnapshot(ctx context.Context, snapshot *snapshots.Snapshot) er
 	return nil
 }
 
-func (s *s3) RestoreSnapshot(ctx context.Context, className, snapshotID string) error {
+func (s *s3) RestoreSnapshot(ctx context.Context, className, snapshotId string) error {
```

Comment:
To be consistent with the rest of the codebase, as well as golang casing convention, I think we should keep this as `snapshotID`